backend/expense/CHANGELOG.md

```
@@ -0,0 +1,13 @@
+
+All notable changes to this module will be documented in this file.
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading
null

(MD041, first-line-heading, first-line-h1)